src/pdn/src/PdnGen.tcl

```diff
@@ -3817,6 +3827,7 @@ proc generate_core_rings {core_ring_data} {
       set inner_ux [expr $xMax - $offset - $spacing - $width]
       set inner_uy [expr $yMax - $offset - $spacing - $width]
     } elseif {[dict exists $layer_info core_offset]} {
+
```

Comment:
Lots of empty lines added for some reason

Reply:
I'll fix it.